### PR TITLE
2D and 1D fixtures on the same bus 

### PR DESCRIPTION
**Pull request by dosipod**

### CODE REVIEW

File: docs/advanced/2d-1d-Mixed-Setup.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: 2D and 1D Mixed Setup
+hide:
+  # - navigation
+  # - toc
+---
+
+## Overview
+
+WLED supports mixing 2D and 1D setup on the same unit, the expected result is that you could use 2D fixture and still chain a strip after or setup a strip and then a matrix after on the same pin.
```
Comment (scottrbailey) [inline docs/advanced/2d-1d-Mixed-Setup.md:10]: > same unit, the expected
Make this two sentences.  "same unit.  The expected"

For the image, you are uploading it.  So set the source to either Aircookie's repo or better yet, a relative URL "/assets/images/content/2D-1D-MIX.png"

Comment (dosipod) [reply docs/advanced/2d-1d-Mixed-Setup.md:10]: okay

**Comment by blazoncek** [comment]
Is this page linked to from main navigation somewhere or does it happen automatically?

**Comment by scottrbailey** [comment]
Actually, this needs linked from the 2D setup page... that to my surprise hasn't been written yet.
So I thought, I'd see what MM has for 2D documentation, and they don't have anything either.  So it looks like we've got the cart before the horse. 

WLED SR has 2D documentation, but we've come a LONG way since that document was written. 
https://github.com/atuline/WLED/wiki/2D-Support

We have a page on mapping, but that will need updated to discuss how 2D is handled. And we need a page on gap files.  ALDIY, if you can focus on mapping and gaps, I'll write the documentation for 2D.

**Comment by dosipod** [comment]
I was not really sure I am allowed to mess with the index page , so i just added it under  WLED-Docs/docs/advanced

**Comment by blazoncek** [comment]
> I was not really sure I am allowed to mess with the index page , so i just added it under WLED-Docs/docs/advanced

why not? please do so, but test using mkdocs first

**Comment by blazoncek** [comment]
Looks like the images got lost.

**Comment by dosipod** [comment]
If you edited to add the note at the end  then i think that changed the last line and  it does not look the same in the KB as in my repo , Last line seems missing spaces  , we can change with other PRs for other docs , also  I think the  note on reversing the segment is not correct as i did not need to do that in my demo in discord  
https://discord.com/channels/473448917040758787/801531333067014164/1145285174498709605
. If you noticed that on your fixture  or just saying **maybe** 

image

**Comment by blazoncek** [comment]
Yes, I added that text directly in GH.
I had to fix missing image as it had relative path instead of absolute.

**Comment by dosipod** [comment]
Yeah , i am still not sure if that is correct , after your changes the image does not show  correctly in gh but will say the word example and it is not a valid link , what is the correct way to do that , i am working on a temp repo and i see the images correctly displaying  with that and other docs  , I should still do all of them the same way as you did ?

image

**Pull request merged**